src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user in ascending alphabetical (case-insensitive) order of names.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a sorted list using alphabetical (case-insensitive) order of names.\n"
+            + "Example: " + COMMAND_WORD;
+
+    /**
+     * Returns a list of persons in sorted alphabetical order according to person names.
+     *
+     * @param immutablePersons list of persons in address book in original order
+     * @return list
+     */
+    private static List<ReadOnlyPerson> sortPersonsByName(List<ReadOnlyPerson> immutablePersons) {
+        List<ReadOnlyPerson> mutablePersons = new ArrayList<>(immutablePersons);
+        Collections.sort(mutablePersons, new Comparator<ReadOnlyPerson>() {
```

Comment:
This looks good. Although you can try to learn about how lambdas can be used to make comparators in one line.

E.g. 

```
mutablePersons.sort((o1, o2) -> o1.getName().fullName.toLowerCase().compareTo(o2.getName().fullName.toLowerCase()));
```

Its not compulsory to change but you can explore it as it is a new feature in Java 8